src/main/java/io/pravega/perf/PravegaStreamHandler.java

```diff
@@ -148,10 +148,9 @@ void recreate() throws InterruptedException, ExecutionException, TimeoutExceptio
         }
     }
 
-    ReaderGroup createReaderGroup(boolean reset) throws URISyntaxException {
+    ReaderGroup createReaderGroup(boolean reset, ClientConfig clientConfig) throws URISyntaxException {
         if (readerGroupManager == null) {
-            readerGroupManager = ReaderGroupManager.withScope(scope,
-                    ClientConfig.builder().controllerURI(new URI(controllerUri)).build());
+            readerGroupManager = ReaderGroupManager.withScope(scope, clientConfig);
```

Comment:
Have you verified that readers in Pravega Benchmark can also read with security enabled?

Reply:
Yes, I have used readers as well. I've added step 3 in the Description section that shows the executed command.